Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -78,75 +108,514 @@ public static void ShowWindow()
+            EditorGUILayout.LabelField("To build applications for AR/VR headsets you need to specify an XR pipeline. "
+                + $"\n\nThe XR SDK pipeline with Unity and Microsoft OpenXR plugins are detected in the project. You are good to go."
+                + $"\n\nFor more information on alternative pipelines, please click on the Learn more button.", EditorStyles.wordWrappedLabel);
+            CreateSpace(15);
+            using (new EditorGUILayout.HorizontalScope())
+            {
+                if (GUILayout.Button("Next"))
+                {
+                    currentStage = ConfigurationStage.ProjectConfiguration;
+                    Repaint();
+                }
+                if (GUILayout.Button("Learn more"))
+                {
+                    Application.OpenURL(XRPipelineDocsUrl);
+                }
 
-            string foldoutHeader;
+            }
+            RenderSetupLaterSection();
+        }
 
-            if (!MixedRealityProjectConfigurator.IsProjectConfigured())
+        private void RenderOpenXRPipelineDetected()
+        {
+            EditorGUILayout.LabelField("XR Pipeline Setting - XR SDK with Unity OpenXR plugin in use", EditorStyles.boldLabel);
+            EditorGUILayout.LabelField("To build applications for AR/VR headsets you need to specify an XR pipeline. "
+                + $"\n\nThe XR SDK pipeline with Unity OpenXR plugin is detected in the project. You are good to go."
+                + $"\n\nNote: If you are targeting HoloLens 2 or HP Reverb G2 headset you need to install the Microsoft OpenXR plugin."
+                + $"\n\nFor more information on alternative pipelines, please click on the Learn more button.", EditorStyles.wordWrappedLabel);
+            CreateSpace(15);
+            using (new EditorGUILayout.HorizontalScope())
             {
-                foldoutHeader = "Modify Configurations";
-                RenderChoiceDialog();
+                if (GUILayout.Button("Next"))
+                {
+                    currentStage = ConfigurationStage.ProjectConfiguration;
+                    Repaint();
+                }
+                if (GUILayout.Button("Install Microsoft OpenXR plugin..."))
+                {
+                    currentStage = ConfigurationStage.InstallMSOpenXR;
+                    Repaint();
+                }
+                if (GUILayout.Button("Learn more"))
+                {
+                    Application.OpenURL(XRPipelineDocsUrl);
+                }
+
             }
-            else
+            RenderSetupLaterSection();
+        }
+
+        private void RenderXRSDKBuiltinPluginPipelineDetected()
+        {
+            EditorGUILayout.LabelField("XR Pipeline Setting - XR SDK with builtin plugin (non-OpenXR) in use", EditorStyles.boldLabel);
+            EditorGUILayout.LabelField("To build applications for AR/VR headsets you need to specify an XR pipeline. "
```

Comment:
this is a bit of a nit, but the line "To build applications for AR/VR headsets you need to specify an XR pipeline. " is reused a lot. Could it be refactored out into a `const` string?

Reply:
Also "\n\nFor more information on alternative pipelines, please click on the Learn more button."